nano/nano_node/entry.cpp

```diff
@@ -716,7 +717,7 @@ int main (int argc, char * const * argv)
 				std::cout.write (reinterpret_cast<const char *> (seed.data.bytes.data ()), seed.data.bytes.size ());
 			}
 		}
-		else if (vm.count ("debug_rpc"))
+		else if (vm.count ("debug_ipc"))
```

Comment:
This change broke a script I have. `--debug_rpc` was a v18 feature so we probably shouldn't change it. I feel `debug_rpc ` is the correct option name here since it receives an RPC json string, not an IPC sequence.

Reply:
ok done